addons/account/models/account_move.py

```diff
@@ -508,7 +508,7 @@ def _sequence_fixed_regex(self):
         help="Is the move being sent asynchronously",
         compute='_compute_is_being_sent'
     )
-
+    move_sent_values = fields.Char(string='Sent', compute='compute_move_sent_values', store=True)
```

Comment:
do we really need a field for that, can't we use the is_move_sent ?

Reply:
The new variable was the only way I found to display a tag as requested by the PO, using is_move_sent I could only figure out a checkbox/toggle or a tag looking like this. Do you know of any way to get around this? Maybe there is a solution I'm not seeing :D 
![image](https://github.com/odoo/odoo/assets/47562677/bf77091a-f72b-4b45-bb9a-37de0e3c8603)

Reply:
I will try to launch the branch in local to see :smile: